### PR TITLE
Add missed changes to goreleaser for brew

### PR DESCRIPTION
**Pull request by J-Thompson12**
**What changed?**
I was able to get a working version of this code tested. The issue I was having was that the releases were binaries and not tar.gz. This was making it so that brew could not find the binary named gitops.

**Why?**

**How did you test it?**
Put this code on a test repo and got the brew install working.

**Release notes**

**Documentation Changes**

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -9,13 +9,13 @@ checksum:
...
   prerelease: auto
 archives:
-  - format: binary
+  - format: tar.gz
     replacements:
       amd64: x86_64
     name_template: "gitops-{{.Os}}-{{.Arch}}"
 builds:
   - <<: &build_defaults
-      binary: "gitops-{{.Os}}-{{.Arch}}"
```
Comment (J-Thompson12) [inline .goreleaser.yml:18]: This line plus the format not being tar.gz was where the problem was.

**Review by jrryjcksn — APPROVED** [review]
:+1:

**Comment by josecordaz** [comment]
I think these changes are going to break this installation guide, right? https://docs.gitops.weave.works/docs/getting-started/#install-the-weave-gitops-cli.

Can we ask David just to make sure this is the desired behavior?

**Comment by J-Thompson12** [comment]
They will break the install guide. I have a different PR for that. These changes wont go into effect until after we do another release. So I have to wait until the release to change the docs.

**Review by josecordaz — APPROVED** [review]
LGTM

**Pull request merged**